modules/ingester/instance.go

```diff
@@ -287,6 +287,10 @@ func (i *instance) CutBlockIfReady(maxBlockLifetime time.Duration, maxBlockBytes
 		}
 
 		completingBlock := i.headBlock
+		blockID := (uuid.UUID)(completingBlock.BlockMeta().BlockID)
+		blockSize := completingBlock.DataLength()
+
+		level.Info(i.logger).Log("msg", "head block cut. enqueueing flush op", "block", blockID, "size", blockSize)
```

Comment:
this log message is not necessarily true. we could return an err on line 308 and the flush op would not be enqueued.

- keep the log message in flush.go that indicates the flush op is enqueued
- change this log message to simply indicate the head block has been cut